lib/Mojolicious/Guides.pod

```diff
@@ -398,6 +398,8 @@ This is the class hierarchy of the L<Mojolicious> distribution.
 
 =item * L<Mojolicious::Command::Author::generate::makefile>
 
+=item * L<Mojolicious::Command::Author::generate::dockerfile>
```

Comment:
These lists look like they're alphabetical, but I wasn't certain if I should really put this between `app` and `lite_app` -- happy to adjust if alphabetical is preferred however. :+1:

Reply:
Yes, always alphabetical.